src/connect-zbt-2/getting-started-with-zigbee/zbt-2-migrating-deCONZ-using-zha.md

```diff
@@ -4,5 +4,96 @@ zendesk:
   name:  Migrating a deCONZ network using Zigbee Home Automation (ZHA)
   position: 40
   labels: connect-zbt-2, getting-started
+productName: "Home Assistant Connect&nbsp;ZBT-2"
+type: "zbt2"
 ---
 
+Follow this guide if you have a deCONZ network running with a Conbee III, a ConBee II or RaspBee II and want to migrate to {{ productName }} and {% abbr "ZHA" %}.
+
+{% callout "note" %}
+
+**Not possible to migrate all settings and devices**
+
+This procedure only helps migrate basic network settings to {{ productName }}.
+
+- Customizations like device names will be lost and automations will need to be updated.
+- Most powered devices like light bulbs will be re-discovered over time (you can speed this up by rebooting the device) but your battery-powered devices may need to be re-joined to the network for the migration to complete.
+- There is currently no migration path to migrate all settings and devices.
+
+{% endcallout %}
+
+This migration includes the following steps:
+
+1. [Prerequisites](#prerequisites).
+2. [Migrating the deCONZ network from the old adapter to ZHA](#to-migrate-a-deconz-network-from-the-old-adapter-to-zha).
+3. [Migrating this ZHA network to {{ productName }}](#to-migrate-the-zha-network-to-connectzbt-2).
+
+## Prerequisites
+
+- a deCONZ network running with a Conbee III, a ConBee II, or a RaspBee II.
+- Home Assistant version 2025.10.1 or later.
+- have the latest firmware installed on the Conbee/Raspbee adapter
+  - check the steps below to see how to [check the version or update the deCONZ adapter](#updating-the-deconz-adapter).
```

Comment:
Per above comment (in ZBT-1 article): The ZBT-1 article starts with an upper-case "Check the steps". The ZBT-2 with a lower-case "check the steps".